index.bs

```diff
@@ -1236,6 +1247,12 @@ To <dfn noexport>create an attribution trigger</dfn> given a [=string=]
 1. Let |debugReportingEnabled| be false.
 1. If |value|["`debug_reporting`"] [=map/exists=] and is a [=boolean=], set
     |debugReportingEnabled| to value["`debug_reporting`"].
+1. Let |aggregationCoordinator| be null.
+1. If |value|["`aggregation_coordinator_identifier`"] [=map/exists=]:
+    1. If |value|["`aggregation_coordinator_identifier`"] is not a [=string=], return null.
+    1. If |value|["`aggregation_coordinator_identifier`"] is not an [=ASCII case-insensitive=] match
```

Comment:
Do want to bring up the case sensitivity question again. It seems that HTML attributes are always case insensitive and javascript enums are always case sensitive. Do we know if there's anything more comparable to this situation to model off of?

One thing that maybe leans towards case sensitivity is that we do exact matches for the JSON keys, right? So maybe it also makes sense to match values exactly.

Reply:
@johnivdel @csharrison for thoughts, thanks!

Reply:
Updated to do a case-sensitive match, thanks!